shell/browser/api/electron_api_web_contents.h

```diff
@@ -211,6 +211,7 @@ class WebContents : public ExclusiveAccessContext,
   void SetDevToolsWebContents(const WebContents* devtools);
   v8::Local<v8::Value> GetNativeView(v8::Isolate* isolate) const;
   void IncrementCapturerCount(gin::Arguments* args);
+  // TODO(codebytere): deprecate stayHidden and stayAwake params.
```

Comment:
Deprecate the API, not just the params ?

Reply:
Feels like we should deprecate `IncrementCapturerCount()` and `DecrementCapturerCount()` altogether?